hoomd/Integrator.cc

```diff
@@ -436,11 +441,11 @@ void Integrator::computeNetForce(uint64_t timestep)
         unsigned int nparticles = m_pdata->getN();
         assert(nparticles <= net_force.getNumElements());
         assert(6*nparticles <= net_virial.getNumElements());
-        for (force_constraint = m_constraint_forces.begin(); force_constraint != m_constraint_forces.end(); ++force_constraint)
+        for (const auto& constraint_force : m_constraint_forces)
             {
-            GlobalArray<Scalar4>& h_force_array =(*force_constraint)->getForceArray();
-            GlobalArray<Scalar>& h_virial_array =(*force_constraint)->getVirialArray();
-            GlobalArray<Scalar4>& h_torque_array = (*force_constraint)->getTorqueArray();
+            GlobalArray<Scalar4>& h_force_array =constraint_force->getForceArray();
```

Comment:
I would like to avoid small change requests like this but I don't know that HOOMD will be able to adopt clang-format in the near term, given the amount of open work...
```suggestion
            GlobalArray<Scalar4>& h_force_array = constraint_force->getForceArray();
```